docs/networks/mainnet/become-a-validator.md

```diff
@@ -107,9 +107,9 @@ If you are setting up a node for the testnet, add the `--testnet` flag to the st
 
 :::
 
-:::note
+:::caution
 
-You will need the LUKSO CLI Version 0.8.0 or above in order to use the `--checkpoint-sync` command. If you are using an older version, please pass down the checkpoint flag as described in the [LUKSO CLI Documentation](https://github.com/lukso-network/tools-lukso-cli/tree/main#using-checkpoint-syncing).
```

Comment:
Shouldn't this still be kept as a mention that the `checkpoint-sync` command is available only from v0.8.0?